src/rcheevos/consoleinfo.c

```diff
@@ -136,7 +142,7 @@ const char* rc_console_name(int console_id)
       return "PC-FX";
 
     case RC_CONSOLE_PC_ENGINE:
-      return "PCEngine";
+      return "PC Engine";
```

Comment:
This was changed and merged 5 months ago. Is your branch out of date?

Reply:
Yeah, apparently it was three commits behind